src/pyedb/grpc/database/primitive/polygon.py

```diff
@@ -272,5 +272,7 @@ def in_polygon(
 
     def add_void(self, polygon):
         if isinstance(polygon, list):
-            polygon = self._pedb.modeler.create_polygon(points=polygon, layer_name=self.layer.name)
-        return self._edb_object.add_void(polygon._edb_object)
+            polygon = self._pedb.modeler.create_polygon(
+                points=polygon, layer_name=self.layer.name, net_name=self.net.name
```

Comment:
Replace self.net.name with self.net_name. The later is faster

Reply:
Done